CodeEdit/Features/UtilityArea/DebugUtility/TaskOutputView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TaskOutputView.swift
+//  CodeEdit
+//
+//  Created by Tommy Ludwig on 27.06.24.
+//
+
+import SwiftUI
+
+struct TaskOutputView: View {
+    @ObservedObject var activeTask: CEActiveTask
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(activeTask.output)
```

Comment:
Would the easiest solution be to use an array of strings instead of a single string? This way, we could use a lazy view, which should solve the beachball issue.

Reply:
That would probably work yeah.